### PR TITLE
Implemented partner module.

### PR DESCRIPTION
**Pull request by dineshGramenerit**
- Implemented Partner list UI screen.
- Integrated all necessary partner list API with its data flow.
- Implemented Delete alert UI & functionality.
- Infinite Scroll API.
- Implemented search API functionality.
- Implemented Export PDF & Excel options.
- Created Add and Edit Partner form UI screens.
- Implemented Multiple skills functionalities.
- Form validations added to the Add and Edit screen.
- Implemented Edit functionality.

### CODE REVIEW

**Review by deviprasad0327 — APPROVED** [review]
Looks good!

File: resource_management_mobile/angular.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
                 "output": "./svg"
               }
             ],
-            "styles": ["src/theme/variables.scss","src/theme/theme.scss", "src/global.scss"],
+            "styles": ["src/theme/variables.scss","src/theme/theme.scss", "node_modules/bootstrap/dist/css/bootstrap.min.css", "src/global.scss"],
```
Comment (premjiGramaner) [inline resource_management_mobile/angular.json:34]: Why do we need boostraop in mobile?

Comment (dineshGramenerit) [reply resource_management_mobile/angular.json:34]: For some styling purposes, we used that.

File: resource_management_mobile/src/app/core/enum/status.enum.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export enum Status {
   INVALID = 'INVALID',
+  SAVE = 'save'
```
Comment (premjiGramaner) [inline resource_management_mobile/src/app/core/enum/status.enum.ts:3]: Is there no camel case here?

**Comment by premjiGramaner** [comment]
@dineshGramenerit and @deviprasad0327 - I need a PR for each task/module. If the branch refers partner then I shouldn't see other module/file changes. This is the last warning If this happens next then I'll reject the PR

**Pull request merged**